templates/http-py/content/app.py

```diff
@@ -1,8 +1,10 @@
-from spin_http import Response
+from spin_sdk.http import simple
+from spin_sdk.http.simple import Request, Response
 
-
-def handle_request(request):
-
-    return Response(200,
-                    {"content-type": "text/plain"},
-                    bytes(f"Hello from the Python SDK", "utf-8"))
+class IncomingHandler(simple.IncomingHandler):
```

Comment:
I thought we exterminated `simple` or did I misunderstand?

Reply:
oh yes - I forgot to update this. I will fix it.

Reply:
fixed it!